src/include/daos_srv/pool.h

```diff
@@ -364,8 +410,7 @@ ds_pool_get_version(struct ds_pool *pool)
 int
 ds_start_chkpt_ult(struct ds_pool_child *child);
 void
-ds_stop_chkpt_ult(struct ds_pool_child *child);
-struct rdb_tx;
+    ds_stop_chkpt_ult(struct ds_pool_child *child);
```

Comment:
this indentation quirk seems to be applied by clang format, I'm not sure why and I have seen it behave intermittently

Reply:
Yeah, this is a bug which I think is with git-clang-format, the issue is the definition is now aligned with the following definition which itself is "wrong" and not modified so not altered.  The simplest fix for this is probably to just correctly align L414 as well.